notebooks/001-getting-started/001-getting-started.ipynb

```diff
@@ -54,10 +54,13 @@
     "\n",
```

Comment:
<p>Maybe some explanation what are the experiment logger and callbacks, and what is the role of the trainer object.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>